src/steamship/base/tasks.py

```diff
@@ -192,6 +198,17 @@ def dict(self, **kwargs) -> Dict[str, Any]:
             }
         return super().dict(**kwargs)
 
+    @classmethod
+    def get(cls, client, _id) -> IResponse[Task]:
```

Comment:
nit: can we type the args of `get`?

also below I think OK to just leave out the `space*` args?